cardano-explorer-db/cardano-explorer-db.cabal

```diff
@@ -30,6 +30,7 @@ library
                         -Wincomplete-uni-patterns
 
   exposed-modules:      Explorer.DB
+                        Test.IO.Explorer.DB.Util
```

Comment:
Still need to fix that as well.

Reply:
i'll get that tomorrow, its getting late here